### DISCUSSION TITLE
Slider size

### DISCUSSION THREAD

**Discussion by LaurenedePeyrelongue** [h2oai/wave#1973] [Q&A] [answered]
Hi,

I am trying to adjust the size of a slider when it is in an ui.inline, however, I can't find any size/width option.
The slider is very small (see image below)
![image](https://github.com/h2oai/wave/assets/88673878/3e233328-6fb5-405a-9561-7de71d89a9d2)

So far the only way I managed to change the slider size is by adding characters to the slider title 
![image](https://github.com/h2oai/wave/assets/88673878/a050e3fe-815a-40a0-919a-6503a17c7a6f)

Is there any way to modify the size?

**Comment by mturoci** [comment, marked as answer]
This is a bug on Wave side. Thanks for reporting! As a workaround, you can use `width` attribute to set the size you want explicitly. E.g.

```py
# Form / Slider
# Use a #slider to allow users to set a value within a specific range.
# #form
# ---
from h2o_wave import main, app, Q, ui


@app('/demo')
async def serve(q: Q):
    q.page['example'] = ui.form_card(box='1 1 4 5', items=[
        ui.inline(items=[
            ui.slider(name='slider', label='Standard slider', min=0, max=100, step=10, value=30, width='100%'),
            ui.slider(name='slider_zero', label='Origin from zero', min=-10, max=10, step=1, value=-3, width='100%'),
            ui.slider(name='slider_disabled', label='Disabled slider', min=0, max=100, step=10, value=30,
                      disabled=True, width='100%'),
        ]),
    ])
    await q.page.save()

```

**Reply by LaurenedePeyrelongue** [reply]
Thanks!
